editor/test/resources/save_data_project/silently_migrated/legacy_render_prototype.render

```diff
@@ -0,0 +1,5 @@
+script: "/builtins/render/default.render_script"
+materials {
+  name: "test"
+  material: "/builtins/materials/sprite.material"
+}
```

Comment:
This also looks suspiciously hand-edited seeing as there is no newline at the end. 🤔